### PR TITLE
Adding accuracy for regression and SHAP explanability

### PR DESCRIPTION
**Pull request by vipgupta**

### CODE REVIEW

File: uptrain/constants.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class AnnotationMethod(Enum):
 
 
 class Monitor(str, Enum):
+    ACCURACY = "accuracy"
```
Comment (sourabhagr) [inline uptrain/constants.py:25]: do we need this?

File: uptrain/constants.py [hunk 2]
```diff
@@ -37,8 +38,9 @@ class Statistic(str, Enum):
 
 
 class Visual(str, Enum):
-    UMAP = "umap"
-    TSNE = "tsne"
+    UMAP = "UMAP"
+    TSNE = "t-SNE"
+    SHAP = "SHAP"
```
Comment (sourabhagr) [inline uptrain/constants.py:43]: Should we add SHAP to visuals?

File: uptrain/constants.py (hunk 1 above)
Comment (vipgupta) [reply uptrain/constants.py:25]: It's getting used. I was thinking we can have an accuracy class where we have all accuracy measures (for classification, regression, etc) are listed.

File: uptrain/constants.py (hunk 2 above)
Comment (vipgupta) [reply uptrain/constants.py:43]: Not sure what's the best place. For now, it's more about visualization, so added it here.

**Review by sourabhagr — APPROVED** [review]

**Pull request merged**